tests/periph_rtt_min/README.md

```diff
@@ -0,0 +1,7 @@
+## About
+
+Describe here what the test application is about
```

Comment:
Maybe "Describe here what the test application is about" ? :)
(and thanks for using `make generate-test` btw ;))